src/controller/CHIPDeviceController.cpp

```diff
@@ -1071,6 +1085,49 @@ DeviceCommissioner::ContinueCommissioningAfterDeviceAttestation(DeviceProxy * de
     return CHIP_NO_ERROR;
 }
 
+#if CHIP_DEVICE_CONFIG_ENABLE_NFC_BASED_COMMISSIONING
```

Comment:
Does this need to be NFC-specific?  In theory this all works for any transport that supports an unpowered phase, right?